browser/src/Editor/NeovimEditor/WelcomeBufferLayer.tsx

```diff
@@ -553,74 +590,74 @@ export interface IWelcomeCommandsViewProps extends Partial<WelcomeViewProps> {
 }
 
 export class WelcomeCommandsView extends React.PureComponent<IWelcomeCommandsViewProps, {}> {
+    public isSelected = (command: string) => command === this.props.selectedId
     public render() {
-        const { commands, executeCommand } = this.props
-        const isSelected = (command: string) => command === this.props.selectedId
+        const { commands } = this.props
         return (
             <LeftColumn>
                 <AnimatedContainer duration="0.25s">
                     <SectionHeader>Quick Commands</SectionHeader>
                     <WelcomeButton
                         title="New File"
-                        onClick={() => executeCommand(commands.openFile.command)}
+                        onClick={() => commands.openFile.execute()}
                         description="Control + N"
                         command={commands.openFile.command}
-                        selected={isSelected(commands.openFile.command)}
+                        selected={this.isSelected(commands.openFile.command)}
                     />
                     <WelcomeButton
                         title="Open File / Folder"
-                        onClick={() => executeCommand(commands.openWorkspaceFolder.command)}
                         description="Control + O"
+                        onClick={() => commands.openWorkspaceFolder.execute()}
                         command={commands.openWorkspaceFolder.command}
-                        selected={isSelected(commands.openWorkspaceFolder.command)}
+                        selected={this.isSelected(commands.openWorkspaceFolder.command)}
                     />
                     <WelcomeButton
                         title="Command Palette"
```

Comment:
This and line 617 need updating.

"Command Palette" -> "Quick Open"
"Control + Shift + P" -> "Ctrl + P".

Reply:
Actually, It may be worth having both, the Command Palette is a little hard to find currently.

It also just made me realise, is it `Ctrl-Shift-P`/`Ctrl-P` on both platforms? We might need to show `cmd` on Mac.

Reply:
This one is intended to be the `command palette` I've just stuck with @bryphe original list for the most part, and on mac `c-s-p` works just as well as `m-s-p`, I think its simpler to leave it as the control binding as its correct for both?

Reply:
Okay!

The command should be updated from `commands.quickOpenShow` to `commands.commandPalette` then I think, as well as changing the actual command being called from `quickOpen` to `command` (lines 355)? Otherwise its saying "Command Palette" and opening a different menu.

(It was like this in the original Welcome screen, so not 100% what @bryphe wanted here, but should probably make the description and action line up at least, can always swap / add what is here later.)

Reply:
LOL I think I only just realised what you meant re. the name changes at @CrossR  I didn't realise the command palette command was opening the quick open 🤣 I thought you meant the names should be different aka that the command palette is actually called the quick open menu, sorry about that 👋